packages/server/graphql/types/TeamMember.ts

```diff
@@ -39,17 +39,26 @@ const TeamMember = new GraphQLObjectType<any, GQLContext>({
       type: GraphQLBoolean,
       description: 'true if the user is a part of the team, false if they no longer are'
     },
-    isLead: {type: GraphQLBoolean, description: 'Is user a team lead?'},
+    isLead: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      description: 'Is user a team lead?',
+      resolve: ({isLead}) => !!isLead
+    },
     isSpectatingPoker: {
       type: GraphQLNonNull(GraphQLBoolean),
       description: 'true if the user prefers to not vote during a poker meeting',
       resolve: ({isSpectatingPoker}) => !!isSpectatingPoker
     },
     hideAgenda: {
       type: new GraphQLNonNull(GraphQLBoolean),
-      description: 'hide the agenda list on the dashboard',
+      description: 'hide the agenda list drawer on the dashboard',
       resolve: ({hideAgenda}) => !!hideAgenda
     },
+    hideManageTeam: {
```

Comment:
instead of `hideAgenda` and `hideManageTeam` could we return `openDrawer: {type: DrawerTypeEnum}`? it should be null if nothing should be open. that way there's no ambiguity on what to show if both are false or both are true